materials/gganimate.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+element: notes
+title: gganimate
+language: R
+---
+
+> ImageMagick needs to be installed to save as gif (the default)
```

Comment:
Should these three lines be bulleted, or maybe separated by new lines? They're showing up as a single block right now.